arb_os/codeSegment.mini

```diff
@@ -342,7 +338,7 @@ public impure func translateEvmCodeSegment(
                     }
                 } else {
                     if (opcode == 0x58) {  // GETPC
-                        return Some((pushValue(restOfCode, restOfCode), evmJumpTable));
+                        return Some((pushValue(bytestream_bytesReadSoFar(bs), restOfCode), evmJumpTable));
```

Comment:
This is tricky so if we're looking to make sure we're compatible it would be nice to have a test. One simple option would be to write a solidity contract which has a function which returns a value based on `GETPC`, run it in geth/ganache in order to figure out what the correct result is, and then run it in ArbOS comparing against that result.

Reply:
Hmm, I think there's an off-by-one error in the code you reviewed.  Which is now fixed.

Earlier today I added an issue to do this kind of testing for EVM->AVM translation more generally.

Reply:
Sounds good